backend/test/core/behaviour/computeGroupBalance.test.ts

```diff
@@ -262,7 +262,56 @@ describe("Calcul group balance (entity)", () => {
           expect(result).toEqual(expectedResult);
         });
       });
+      describe("Given a group of two members that with two transactions", () => {
+        beforeEach(() => {
+          members = [new GroupMember(0, "Luc"), new GroupMember(1, "Jessica")];
+        });
+        test("with transaction 1 :payerId 1 and amount 3 for whole group, transaction 2 :payerId 1 and amount 1 for member 0, it should return a balance {0 :-2.5,1 :2.5}.", async () => {
+          // Arrange
+          const group = new Group(1, "group", members, [
+            new Transaction(1, 1, [0, 1], 3),
+            new Transaction(2, 1, [0], 1),
+          ]);
+
+          // Act
+          const result = computeGroupBalance(group);
+
+          // Assert
+          const expectedResult = new GroupBalance(
+            1,
+            new Map<number, number>([
+              [0, -2.5],
+              [1, 2.5],
+            ])
+          );
+          expect(result).toEqual(expectedResult);
+        });
+      });
+      describe("Given a group of two members with two transactions, membersId starting at 1", () => {
+        beforeEach(() => {
+          members = [new GroupMember(1, "Luc"), new GroupMember(2, "Jessica")];
+        });
+        test("with transaction 1 :payerId 1 and amount 3 for whole group, transaction 2 :payerId 1 and amount 1 for member 0, it should return a balance {0 :-2.5,1 :2.5}.", async () => {
```

Comment:
Not really readable and I don't understand what you mean with the balance you are supposed to get (error in the indexes ?)